src/FileService/Controllers/FileController.cs

```diff
@@ -36,13 +34,14 @@ public class FileController : ControllerBase
       return result.Select(file => File(file.content, file.extension, file.name)).ToList();
     }
 
-    [HttpPatch("edit")]
+    [HttpPut("edit")]
     public async Task<OperationResultResponse<bool>> EditAsync(
       [FromServices] IEditFileCommand command,
+      [FromQuery] Guid entityId,
       [FromQuery] Guid fileId,
-      [FromBody] JsonPatchDocument<EditFileRequest> request)
+      [FromQuery] string newName)
```

Comment:
I'd choose to use everywhere newName or just name

Reply:
done